domain/api/atrium-domain-api-common/src/main/kotlin/ch/tutteli/atrium/domain/creating/ComparableAssertions.kt

```diff
@@ -29,4 +29,9 @@ interface ComparableAssertions {
         subjectProvider: SubjectProvider<T1>,
         expected: T2
     ): Assertion
+
+    fun <T1 : Comparable<T2>, T2 : Any?> isEqualComparingTo(
+        expect: Expect<T1>,
```

Comment:
Just because it is your first OS contribution: 
some actions failed: https://github.com/robstoll/atrium/pull/501/checks?check_run_id=687077604#step:4:39
seems like Expect was not imported in this file yet (one of the reasons why the build fails)